.github/workflows/haskell.yml

```diff
@@ -172,9 +168,13 @@ jobs:
       run: cabal build all
 
     - name: Run unit tests
+      shell: bash
       run: |
         cabal test cardano-addresses
-        cabal test cardano-addresses-cli
+        if [ "${{ runner.os }}" != "Windows" ] && [ "${{ matrix.ghc }}" !=  "9.6.2" ]; then
+          # This test is currently broken on windows (unhandled PEi386 relocation type 14) so disable it.
+          cabal test cardano-addresses-cli
+        fi
```

Comment:
I _think_ this is fixed in recent haskell.nix versions.

Reply:
Can also be moved to a separate issue after this is merged and we can move the repository forward?

Reply:
This is not doing anything since `"9.6.2" != "9.6.5"`

Reply:
I've changed this back.

Reply:
It turns out it was still filtering out the test as the `&&` should have been an `||`.  So it was disabling all the test for windows GHC versions (including `"9.6.5"`).  I have put it back with:
* `||` instead of `&&`
* `9.6.5` instead of `9.6.2`
* A comment as I think the failure is related to encoding of emoji's in the output.